guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack

```

Comment:
How about we just go with '#' for top level /in the current file/, ignoring what the website does, then use a filter in md-render to normalize it?